### PR TITLE
Update documentation for v2.1 and change version in master

### PR DESCRIPTION
**Pull request by danielwilms**

### CODE REVIEW

File: docs-gen/content/releases.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ weight: 2
 
 {{< directoryindex path="/static/releases/nightly" pathURL="/vehicle_signal_specification/releases/nightly/" >}}
 
+### Version 2.0
```
Comment (erikbosch) [inline docs-gen/content/releases.md:11]: shall be 2.1?

File: docs-gen/content/releases.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ weight: 2
 
 {{< directoryindex path="/static/releases/nightly" pathURL="/vehicle_signal_specification/releases/nightly/" >}}
 
+### Version 2.1
+
+{{< directoryindex path="/static/releases/v2.0" pathURL="/vehicle_signal_specification/releases/v2.1/" >}}
```
Comment (erikbosch) [inline docs-gen/content/releases.md:13]: My apologies, I did not review thoroughly last time, we have a "v2.0" on this line as well in path

Comment (danielwilms) [reply docs-gen/content/releases.md:13]: I mean, I made the mistake :) I'll push an update

**Review by SebastianSchildt — APPROVED** [review]
I can confirm, the generated JSON seems to work. I checked that e.g. the OBD repairs are in. The Service data points did not make the cut for 2.1, but that is fine for me, if intended, as I am sure 2.2. is just around the corner 😼

**Comment by danielwilms** [comment]
fix the typo, tag & merge

**Pull request merged**